BlueprintUI/Sources/Layout/GeometryReader.swift

```diff
@@ -48,8 +48,8 @@ public struct GeometryProxy {
     /// The size constraint of the element being laid out.
     public var constraint: SizeConstraint
 
-    /// Measure the given element, using this proxy's constraint and the current environment.
-    public func measure(element: Element) -> CGSize {
-        element.content.measure(in: constraint, environment: environment)
+    /// Measure the given element, using the provided constraint (or the proxy's constraint if not provided), and the current environment.
```

Comment:
I’d maybe flip the precedence of the comments here to guide people toward the more-common approach. Maybe something like:
```
/// Measure the given element, constrained to the same size as the `GeometryProxy` itself (unless a constraint is explicitly provided).
```
(I also dropped the environment part because I think it's more of an expected implementation detail than something consumers need be concerned with)

Reply:
Done